operations/validate_csm_health.md

```diff
@@ -461,66 +552,78 @@ ncn# /opt/cray/tests/integration/csm/barebonesImageTest --xname x3000c0s10b4n0
+The procedures below use the CLI as an authorized user and run on two separate node types. The first part runs on the LiveCD node, while the second part runs on a non-LiveCD Kubernetes master or worker node.
+When using the CLI on either node, the CLI configuration needs to be initialized and the user running the procedure needs to be authorized.
 
 The following procedures run on separate nodes of the system.
 
 1. [Validate Basic UAS Installation](#uas-uai-validate-install)
 1. [Validate UAI Creation](#uas-uai-validate-create)
+1. [Test UAI Gateway Health](#test-uai-gateway-health)
 1. [UAS/UAI Troubleshooting](#uas-uai-validate-debug)
    1. [Authorization Issues](#uas-uai-validate-debug-auth)
    1. [UAS Cannot Access Keycloak](#uas-uai-validate-debug-keycloak)
    1. [UAI Images not in Registry](#uas-uai-validate-debug-registry)
    1. [Missing Volumes and Other Container Startup Issues](#uas-uai-validate-debug-container)
 
 <a name="uas-uai-validate-install"></a>
-### 5.1 Validate the Basic UAS Installation
+
+### 6.1 Validate the Basic UAS Installation
 
 This section can be run on any NCN or the PIT node.
 
 1. Basic UAS installation is validated using the following:
    1.
+
       ```bash
       ncn# cray uas mgr-info list
       ```
 
       Expected output looks similar to the following:
+
       ```text
       service_name = "cray-uas-mgr"
       version = "1.11.5"
       ```
 
       In this example output, it shows that UAS is installed and running the `1.11.5` version.
    1.
+
       ```bash
       ncn# cray uas list
       ```
 
       Expected output looks similar to the following:
+
       ```text
       results = []
       ```
 
      This example output shows that there are no currently running UAIs. It is possible, if someone else has been using the UAS, that there could be UAIs in the list. That is acceptable too from a validation standpoint.
 1. Verify that the pre-made UAI images are registered with UAS
+
    ```bash
    ncn# cray uas images list
    ```
 
    Expected output looks similar to the following:
+
    ```text
    default_image = "registry.local/cray/cray-uai-sles15sp3:1.0.11"
    image_list = [ "registry.local/cray/cray-uai-sles15sp3:1.0.11",]
    ```
 
-   This example output shows that the pre-made end-user UAI image (`cray/cray-uai-sles15sp3:1.0.11`) is registered with UAS. This does not necessarily mean this image is installed in the container image registry, but it is configured for use. If other UAI images have been created and registered, they may also show up here, which is acceptable.
+   This example output shows that the pre-made end-user UAI image (`cray/cray-uai-sles15sp3:1.0.11`) is registered with UAS. This does not necessarily mean this image is installed in the container image
```

Comment:
And I guess that would also involve slightly updating this paragraph.

Reply:
done